cli/lib/cli/configuration/repo_configuration.rb

```diff
@@ -19,11 +19,11 @@ def initialize(
   end
 
   def backup_prefs_repo_location
-    @backup_prefs_repo_location || File.join(@backups_location_base, @ide_name)
+    @backup_prefs_repo_location || Dir[File.join(@backups_location_base, @ide_name)].first
```

Comment:
Can you abstract this into a method? There's a bit of duplication here between the two methods (the Dir[...].first is in both), and if you put it behind an intention revealing method name, it will be more obvious what this code is doing.